### PR TITLE
Add constructors and methods to allow managed calls to be able to cre…

### LINKED ISSUE #174: ContestDescription missing party ids
## Bug Report

**Expected Behavior**
Contest descritpions include party id's when necessary

**Actual Behavior**
party id's are not included. there is a missing field.

**Environment**
- OS: [e.g. OSX 10.13.4, Windows 10]
- Version: [e.g. 1.0.1]

### PR DESCRIPTION
**Pull request by SteveMaier-IRT**
### Issue
*Link your PR to an issue*

Fixes #174 

### Description
Added constructors for .net that include the party ids.  This was missing in C# but was already available in C++.

### CODE REVIEW

**Review by keithrfung — CHANGES_REQUESTED** [review]
PR still failing build

File: bindings/netframework/ElectionGuard.NetFramework/ElectionGuardInterop/Manifest.h [hunk 1]
```diff
@@ -1199,8 +1275,7 @@ namespace ElectionGuardInterop
 
             this->_instance = new electionguard::Manifest(
               _electionScopeId, static_cast<electionguard::ElectionType>(type), nativeStartDate,
-              nativeEndDate,
-              std::move(_gpUnits), std::move(_parties), std::move(_candidates),
+              nativeEndDate, std::move(_gpUnits), std::move(_parties), std::move(_candidates),
```
Comment (keithrfung) [inline bindings/netframework/ElectionGuard.NetFramework/ElectionGuardInterop/Manifest.h:1278]: This looks like just a spacing change? That seems weird.

File: bindings/netframework/ElectionGuard.NetFramework/ElectionGuardInterop/Manifest.h [hunk 2]
```diff
@@ -1325,7 +1400,7 @@ namespace ElectionGuardInterop
             return this->_instance->isValid();
         }
 
-          property array<Byte> ^
+        property array<Byte> ^
```
Comment (keithrfung) [inline bindings/netframework/ElectionGuard.NetFramework/ElectionGuardInterop/Manifest.h:1403]: White space change?

**Review by keithrfung — APPROVED** [review]
This looks good but I would remove those white space and spacing changes if it isn't autoformatted by a tool. Seems odd?

Double check approval with @AddressXception

File: bindings/netframework/ElectionGuard.NetFramework/ElectionGuardInterop/Manifest.h (hunk 2 above)
Comment (SteveMaier-IRT) [reply bindings/netframework/ElectionGuard.NetFramework/ElectionGuardInterop/Manifest.h:1403]: That line was not changed, so not sure why the whitespace changed.

**Pull request merged**